crates/trigger/src/cli/stdio.rs

```diff
@@ -95,6 +103,39 @@ impl<F: RuntimeFactors, U> ExecutorHooks<F, U> for StdioLoggingExecutorHooks {
         configured_app: &spin_factors::ConfiguredApp<F>,
     ) -> anyhow::Result<()> {
         self.validate_follows(configured_app.app())?;
+
+        if self.refresh_log {
+            let sanitized_component_ids: Vec<String> = configured_app
+                .app()
+                .components()
```

Comment:
Did we confirm that `.components()` returns _only_ the components being run by _this_ trigger?

Reply:
Yes, returns only this trigger

Reply:
Excellent!